<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>136</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Date0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Time0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDate0" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Date0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime0" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Time0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime1" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Time1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>100</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Time1</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDate1" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Date1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Date1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>130</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>Price Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPriceMax" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>130</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Price0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>200</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Volume</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolumeSum" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPriceMin" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Price0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>150</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>Price Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>170</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Price Dif,%</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPriceDif" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Price0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolumeSup" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>220</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>240</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Vol Dem</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolumeDem" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>240</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>220</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Vol Sup</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Profit</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelProfit" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>270</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>290</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Dens Sup</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>310</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Dens Dem</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDensitySup" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>290</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDensityDem" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>310</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
